modules/installation-uninstall-aws.adoc

```diff
@@ -7,27 +7,87 @@
 
 You can remove a cluster that you installed on Amazon Web Services (AWS).
 
+.Prerequisites
+
+* Deploy an {product-title} cluster.
+* Have a copy of the installation program that you used to deploy the cluster.
+
 .Procedure
 
-. Optionally, from the computer that you used to install the cluster, run the
-following command and record the UUID that it outputs:
+Perform the following steps on the computer that you used to install the cluster.
+
+. Obtain the cluster UUID:
+
+** If you have access to the `oc` command line interface (CLI), log in and run the
```

Comment:
How about:
s/If you have access to the `oc` command line interface (CLI)/If you have access to the running cluster via the `oc` command line interface (CLI)/